src/js/views/patients/shared/components/owner_component.js

```diff
@@ -51,40 +57,57 @@ export default Droplist.extend({
         };
       },
       isSelectlist: true,
-      headingText: i18n.headingText,
```

Comment:
this issue is that these need to be `headingText: this.getOption('headingText'),`. @luketlancaster